store/postgres/src/store.rs

```diff
@@ -434,6 +434,104 @@ impl Store {
 }
 
 impl StoreTrait for Store {
+    fn authorize_subgraph_name(&self, name: String, new_access_token: String) -> Result<(), Error> {
```

Comment:
We're putting a bit much into the `Store`. At some point it will be worth thinking about splitting this up into separate traits and separate implementations. But not right now.

Reply:
Agreed. We should have a separate store component for subgraph name info